php/templates/containers.twig

```diff
@@ -5,683 +5,20 @@
-                    {% else %}
-                        {% if timezone == "" %}
-                            <p>To get the correct time values for certain Nextcloud features, set the timezone for Nextcloud to the one that your users mainly use. Please note that this setting does not apply to the mastercontainer and any backup option.</p>
-                            <p>You can configure the timezone for Nextcloud below:</p>
-                            <form method="POST" action="/api/configuration" class="xhr">
-                                <input type="text" id="timezone" name="timezone" placeholder="Europe/Berlin" />
-                                <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
-                                <input type="hidden" name="{{csrf.keys.value}}" value="{{csrf.value}}">
-                                <input type="submit" value="Submit timezone" onclick="return confirm('Are you sure that this is a valid timezone? Please double check by following the wikipedia article and checking the correct column. If the timezone is not valid, it will break the startup since the database will not be correctly initialized and you will end up in a startup loop.')" />
-                            </form>
-                            <p>You need to make sure that the timezone that you enter is valid. An example is <strong>Europe/Berlin</strong>. You can get valid values by looking at the 'TZ identifier' column of this list: <a href="https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List"><strong>click here</strong></a>. The default is <strong>Etc/UTC</strong> if nothing is entered.</p>
-                        {% else %}
-                            <p>The timezone for Nextcloud is currently set to <strong>{{ timezone }}</strong>. You can change the timezone by clicking on the button below.</p>
-                            <form method="POST" action="/api/configuration" class="xhr">
-                                <input type="hidden" name="delete_timezone" value="yes"/>
-                                <input type="hidden" name="{{csrf.keys.name}}" value="{{csrf.name}}">
-                                <input type="hidden" name="{{csrf.keys.value}}" value="{{csrf.value}}">
-                                <input type="submit" value="Reset the timezone" />
-                            </form>
-                        {% endif %}
-                    {% endif %}
-                {% endif %}
-            {% endif %}
-        {% endif %}
-
-    {% if isApacheStarting == true or is_backup_container_running == true or isWatchtowerRunning == true or is_daily_backup_running == true %}
-        <script type="text/javascript" src="automatic_reload.js"></script>
-    {% else %}
-        <script type="text/javascript" src="before-unload.js"></script>
-    {% endif %}
-
-    </main>
+    <div class="container">
+        {{ include('includes/containers-main.twig') }}
+    </div>
```

Comment:
lets please do this in a follow-up PR to keep the changes minimal and related :)

Reply:
OK. :+1: